.github/workflows/build.yaml

```diff
@@ -65,15 +65,15 @@ jobs:
       - uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
       - name: Build and (optionally) push container image
         id: build
-        uses: stackabletech/actions/build-container-image@ab97c67d79a79d9c6b0dc81b5dbe2fc5fd216a5a # v0.0.4
+        uses: stackabletech/actions/build-container-image@f589fce30aa5c1de94aa22d4e8e9b4b7f0810d40 # vx.x.x
```

Comment:
Are those `vx.x.x` intentional?

Reply:
Yes, it was before the fixed action was merged so there was no version number (yet).
Resolved in 664374c